checker/src/org/checkerframework/checker/fenum/qual/FenumBottom.java

```diff
@@ -1,14 +1,18 @@
 package org.checkerframework.checker.fenum.qual;
 
-import java.lang.annotation.*;
-
-import com.sun.source.tree.Tree;
-
 import org.checkerframework.framework.qual.DefaultFor;
-import org.checkerframework.framework.qual.TypeUseLocation;
 import org.checkerframework.framework.qual.ImplicitFor;
 import org.checkerframework.framework.qual.SubtypeOf;
 import org.checkerframework.framework.qual.TargetLocations;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+import java.lang.annotation.Documented;
```

Comment:
The order in the configuration file is specified via the integer at the beginning of each line rather than the order of the file.  So, org.checkerframework.* is before java.*